scripts/Invoke-SasDeveloperWorkstation.py

```
@@ -0,0 +1,286 @@
+        print((run_root / "english-summary.txt").read_text(encoding="utf-8"), end="")
+        return 0
+    steps: list[dict] = []
+    artifacts: list[dict] = []
+    if platform == "macos" or domain == "unsupported":
+        steps.append(step("platform", "SKIP", "macOS is unsupported by the workstation v3 contract"))
+        unsupported = True
+        agent_result = None
+    else:
+        unsupported = False
+        inventory, inventory_step = run_inventory(platform, fixture, run_root, args.timeout_seconds)
+        steps.append(inventory_step); artifacts.append({"role": "inventory", "path": "inventory.json"})
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)